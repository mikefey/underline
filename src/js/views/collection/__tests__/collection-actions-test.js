import test from 'ava';
import {
  downloadBook,
  getCollection,
  updateCollection,
} from './../collection-actions';

test('downloadBook action', (t) => {
  t.snapshot(downloadBook('/path/to/book'));
});

test('getCollection action', (t) => {
  t.snapshot(getCollection());
});

test('updateCollection action', (t) => {
  const data = { id: '12345', title: 'A New Book' };
  t.snapshot(updateCollection(data));
});
