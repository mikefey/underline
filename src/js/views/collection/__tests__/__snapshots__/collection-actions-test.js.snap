// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collection-actions 1`] = `"{"type":"DOWNLOAD_BOOK","url":"/path/to/book"}"`;

exports[`downloadBook action 1`] = `
Object {
  "type": "DOWNLOAD_BOOK",
  "url": "/path/to/book",
}
`;

exports[`getCollection action 1`] = `
Object {
  "type": "GET_COLLECTION",
}
`;

exports[`updateCollection action 1`] = `
Object {
  "data": Object {
    "id": "12345",
    "title": "A New Book",
  },
  "type": "ADD_BOOK_TO_COLLECTION",
}
`;
